scripts/hostapd/install

```diff
@@ -107,3 +107,8 @@ sudo cp "$BASE_DIR/hostapd.service" /etc/systemd/system/hostapd.service
 sudo systemctl daemon-reload
 sudo systemctl enable hostapd.service
 sudo systemctl start hostapd.service
+
+if [[ -z "$(cat /etc/rc.local | grep dnsmasq)" ]]; then
+	sudo sed -s 's/exit 0/service dnsmasq restart\nexit 0/' /etc/rc.local 
```

Comment:
1. `-s` or `-i`?
1. We are not undoing this if we run uninstall, that's okay?
1. Spaces not tabs. Please.